composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "remp/crm-application-module",
+  "name": "vasyaxy/crm-application-module",
```

Comment:
You probably didn't want to push this change. At least we cannot merge it this way :).